playkit/src/main/java/com/kaltura/playkit/drm/DeferredDrmSessionManager.java

```diff
@@ -123,6 +138,35 @@ public DrmSession acquireSession(Looper playbackLooper, @Nullable EventDispatche
         return drmSessionManager.acquireSession(playbackLooper, eventDispatcher, format);
     }
 
+    private DrmSessionManager getDRMSessionManager(DrmCallback drmCallback) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
+            DefaultDrmSessionManager.Builder drmSessionManagerBuilder = new DefaultDrmSessionManager.Builder();
+            drmSessionManagerBuilder.setMultiSession(true) // key rotation
+                    .setPlayClearSamplesWithoutKeys(allowClearLead);
+
+            if (isForceWidevineL3Playback) {
+                drmSessionManagerBuilder.setUuidAndExoMediaDrmProvider(
+                        MediaSupport.WIDEVINE_UUID,
+                        uuid -> {
+                            try {
+                                FrameworkMediaDrm frameworkMediaDrm = FrameworkMediaDrm.newInstance(MediaSupport.WIDEVINE_UUID);
+                                frameworkMediaDrm.setPropertyString(SECURITY_LEVEL_PROPERTY, WIDEVINE_SECURITY_LEVEL_3);
+                                return frameworkMediaDrm;
+                            } catch (UnsupportedDrmException e) {
+                                throw new IllegalStateException(e);
```

Comment:
@GouravSna 
we better set 
```
drmSessionManager = DrmSessionManager.getDummyDrmSessionManager();
```

instead no?

Reply:
drmSessionManager has already been created at this time because this try/catch is callback from ExoMediaDrm.Provider$acquireExoMediaDrm(UUID uuid).

Reply:
Fixed. Returning the default FrameworkMediaDrm